src/Microsoft.HttpRepl/Preferences/WellKnownPreference.cs

```diff
@@ -172,6 +172,8 @@ public static IReadOnlyList<string> Names
 
         public static string SwaggerSearchPaths { get; } = "swagger.searchPaths";
 
+        public static string SwaggerUIEndpoint { get; } = "swagger.uiEndpoint";
```

Comment:
Is this appropriate as a user setting?  It seems like it would vary from project to project, even though conventions may bring some level of consistency...

Reply:
Discussed offline. Going to leave it as a setting to provide a balance between having to pass it as a parameter to the `ui` command every time and having it always use the `swagger` relative uri. Would be nice if it was specified somewhere in the swagger/openapi definition, but it is not.